front-end/src/app/reports/f1m/main-form/main-form.component.html

```diff
@@ -251,6 +251,12 @@ <h2>Affiliation</h2>
             <div class="field">
               <label for="name">AFFILIATED DATE</label>
               <p-calendar
+                (onFocus)="calendarOpened = true"
```

Comment:
could you make this an onFocus method in the component

Reply:
I left onFocus alone